index.html

```diff
@@ -0,0 +1,188 @@
+                    <p id="text-speakers"></p>
+                </div>
+            </div>
+            <br>
+            <div id="content-members">
+                <div><img id="image-project1" src="." alt="">
+                    <img id="image-project11" src="." alt=""></div>
+                <div>
+                    <h3 id="title-main-speakers1"></h3>
+                    <h5 id="subtitle-main-speakers1"></h5>
+                    <hr class="line2">
+                    <p id="text-speakers1"></p>
+                </div>
+            </div> 
+            <br>
+            <div id="content-members" class="membersDavid">
+                <div><img id="image-project2" src="." alt="">
+                    <img id="image-project22" src="." alt=""></div>
+                <div>
+                    <h3 id="title-main-speakers2"></h3>
+                    <h5 id="subtitle-main-speakers2"></h5>
+                    <hr class="line2">
+                    <p id="text-speakers2"></p>
+                </div>
+            </div>
+            <div id="content-members" class="members">
+                <div><img id="image-project3" src="." alt="">
+                    <img id="image-project33" src="." alt=""></div>
+                <div>
+                    <h3 id="title-main-speakers3"></h3>
+                    <h5 id="subtitle-main-speakers3"></h5>
+                    <hr class="line2">
+                    <p id="text-speakers3"></p>
+                </div>
+            </div>
+            <div class="button-more-main">
+                <a id="button-more" href="#speakers">More</a>
+            </div>
+        </div>
+        <div id="container-partners">
+            <div id="partner-title"><h3 id="title-second">Social Media and Partners</h3></div>
+            <hr class="line5">
+            <div class="container-ul-partners">
+                <div id="partner-objects">
+                    <ul id="classes-photos-partners">
+                        <li><img class="image-partners" src="images+/gorjeo (1).png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" src="images+/tik-tok.png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" src="images+/facebook(1).png" alt=""></li>
+                    </ul>
+                </div>
+                <div id="partner-objects-two">
+                    <ul id="classes-photos-partners-two">
+                        <br>
+                        <li><img class="image-partners" id="image-partners-one" src="images+/instagram.png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" id="image-partners-two" src="images+/whatsapp.png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" src="images+/youtube.png" alt=""></li>
+                    </ul>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly make sure to hide the partner's container on home page page of mobile version in order to stick to the provided design.

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191821467-0f587a98-0f15-4b9f-b73d-6e5dc21076d4.png)


   - ### Behance design
   ![image](https://user-images.githubusercontent.com/87197412/191821407-58c7250e-a4c2-42c8-ad4e-5896fb17a1c4.png)